blynklib_mp.py

```diff
@@ -5,6 +5,7 @@
 __version__ = '0.2.6'
 
 import usocket as socket
+import ussl as ssl
```

Comment:
ussl import can affect library usage on
some micropython ports ..
I suggest create separate file
blynklib_mp_ssl.py

Reply:
I'm not sure about another split. I mean I think even the first split was a bad idea because the common code never gets in sync anymore.

What about I import it inline in `_get_socket()` just before using it?

Reply:
Or.. I can have the new `blynklib_mp_ssl.py` which has a new `class SSLConnection` which inherit from `class Connection`.

The compromise is that people using the SSL version will need to upload 2 files instead of 1. Is that bad?

Reply:
I just pushed new commits showing the concept with the new `SslConnection` class.

Blynk can still be used as usual for clear text transmissions. with no additional imports or anything.

For SSL, it is used like so:

```python
import blynklib_mp as blynklib
import blynklib_mp_ssl as blynklib_ssl

ssl_connection = blynklib_ssl.SslConnection(token, port=443, log=print)
blynk = blynklib.Blynk(secret.BLYNK_AUTH, connection=ssl_connection)
```

We also don't need to switch on the port used.

What do you think?